manim/utils/cfg_subcmds.py

```diff
@@ -22,10 +22,59 @@
 RICH_COLOUR_INSTRUCTIONS = """[red]The default colour is used by the input statement.
 If left empty, the default colour will be used.[/red]
 [magenta] For a full list of styles, visit[/magenta] [green]https://rich.readthedocs.io/en/latest/style.html[/green]"""
-
+RICH_NON_STYLE_ENTRIES = ["log.width", "log.height", "log.timestamps"]
 console = Console()
 
 
+def value_from_string(value):
+    """Extracts the literal of proper datatype from a string.
+    Parameters
+    ----------
+    value : :class:`str`
+        The value to check get the literal from.
+    Returns
```

Comment:
empty line for good measure
```suggestion

    Returns
```